mono-dotnet-node-xvfb/Dockerfile

```diff
@@ -11,8 +11,18 @@ RUN apt-get -q update && \
     liblttng-ust0 \
     libssl1.0.2 \
     libstdc++6 \
-    zlib1g \
-    && rm -rf /var/lib/apt/lists/* /tmp/*
+    zlib1g && \
+# Install OpenJDK-8 && Fix certificate issues
+    apt-get install -y openjdk-8-jdk && \
```

Comment:
why repeating apt-get insall ?
Just add the packages you want to install to the previous list